### DISCUSSION TITLE
"Is there any information in the documentation about email verification for user registration?"

### DISCUSSION THREAD

**Discussion by YANGJIAXUE2022510** [filamentphp/filament#8342] [General]
During registration, my code is throwing an error: Route [filament.admin.auth.email-verification.verify] not defined.
But I can't find any information about email verification or registration in the documentation. Can you provide a hint?
"I just added ->registration() in the code without any other actions. I can't find relevant content."
![image](https://github.com/filamentphp/filament/assets/105269516/64adfd3c-8c52-4d11-9365-d3c1d5b9f6db)
![image](https://github.com/filamentphp/filament/assets/105269516/61661aed-e6cb-4d7c-a807-0f7f4489d86d)

**Comment by YANGJIAXUE2022510** [comment]
I found it in the documentation.   ->emailVerification()   😭

**Reply by NourEdin** [reply]
@hackie-me I know it's late, but you need to run a queue worker (ie: `php artisan queue:work`) because filament send emails in a job.
